.github/workflows/pr.yaml

```diff
@@ -153,6 +153,10 @@ jobs:
       - verify-analyze-code
     runs-on: ubuntu-latest
     steps:
+      - name: Discover init
+        run: |
+          ps -p 1 -o comm=
+          stat /sbin/init
```

Comment:
Trying understand what can be causing: https://github.com/aquasecurity/tracee/actions/runs/6074223465/job/16479631240?pr=3305#step:4:411

Reply:
This will be handled via https://github.com/aquasecurity/tracee/issues/3440